app/src/controllers/step/actionValidators.js

```diff
@@ -17,3 +17,10 @@ export const validateIncludeInAA = (item) => {
   }
   return null;
 };
+
+export const validateLinkIssue = (item) => {
```

Comment:
I think it's pretty simple function for make it in one string:
`export const validateLinkIssue = (item) => !item.issue ? 'noDefectTypeToLinkIssue' : null;`

Reply:
Done